src/pages/[platform]/build-a-backend/storage/storage-v5-to-v6-migration-guide/index.mdx

```diff
@@ -0,0 +1,1265 @@
+        CacheControl?: string;
+        ContentDisposition? string;
+        ContentEncoding?: string;
+        ContentMD5?: string;
+        ContentType?: string;
+        Expires?: Date;
+        Metadata?: Record<string, string>;
+        ServerSideEncryption?: string; // Not supported in v6
+        SSECustomerAlgorithm?: string; // Not supported in v6
+        SSECustomerKey?: string; // Not supported in v6
+        SSECustomerKeyMD5?: string; // Not supported in v6
+        SSEKMSKeyId?: string; // Not supported in v6
+        Bucket: string;
+        Key: string;
+        Body?: string | ReadableStream<any> | Blob | Uint8Array | Buffer;
+        Tagging?: string; // Not supported in v6
+      }
+      // non-resumable config
+      config?: {
+        level?: 'public' | 'protected' | 'private';
+        provider?: string;
+        customPrefix?: {
+          private?: string;
+          public?: string;
+          protected?: string;
+        };
+        track?: boolean; // Not yet supported in V6
+        progressCallback?: (progress: any) => any;
+        serverSideEncryption?: string; // Not supported in v6
+        SSECustomerAlgorithm?: string; // Not supported in v6
+        SSECustomerKey?: string; // Not supported in v6
+        SSECustomerKeyMD5?: string; // Not supported in v6
+        SSEKMSKeyId?: string; // Not supported in v6
+        acl?: string; // Not supported in v6
+        bucket?: string;
+        cacheControl?: string;
+        contentDisposition?: string;
+        contentEncoding?: string;
+        contentType?: string;
+        expires?: Date;
+        metadata?: Record<string, string>;
+        tagging?: string; // Not supported in v6
+        useAccelerateEndpoint?: boolean;
+        resumable?: boolean;
+      }
+      // resumable config
+      config?:  {
+        // All of the standard config above plus
+        resumable: true; // More specific than resumable above
+        progressCallback?: (progress: {
+          loaded: number;
+          total: number;
+        }) => any; // More specific than progressCallback above
+        completeCallback?: (event: { key?: string }) => any;
+        errorCallback?: (err: any) => any;
+      }
+      ```
+
+    </div>
+    <div>
+      **V6**
+
+      ```
+      input: {
+        key: string;
+        data: string | Blob | ArrayBufferView | ArrayBuffer;
+        options?: {
+          accessLevel?: any;
+          useAccelerateEndpoint?: boolean;
+          onProgress?: (event: {
+            transferredBytes: number;
+            totalBytes?: number;
+          }) => void;
+          contentDisposition?: string;
+          contentEncoding?: string;
+          contentType?: string;
+          metadata?: Record<string, string>;
+        };
+      }
+      ```
+    </div>
+  </Columns>
+</Accordion>
+
+<Accordion title='Output' headingLevel='3'>
+  <Columns columns={2}>
+    <div>
+      **V5**
+
+      ```
+      S3ProviderGetPropertiesOutput {
```

Comment:
QQ: The output in v5 `put` was a getProperties one? 🤯

Reply:
Oh I'm wondering if I accidentally edited the wrong one when I was adding the getProperties example? I will take a look, good catch.